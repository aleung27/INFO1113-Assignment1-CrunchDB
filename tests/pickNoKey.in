PICK j 0
SET j 8
PICK J 0
PICK j 1
LIST ENTRIES
