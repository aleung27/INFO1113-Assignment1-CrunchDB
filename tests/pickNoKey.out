> no such key

> ok

> no such key

> 8

> j [8]

